### PR TITLE
cmd, swarm: enforce mixed caps variable names

### PR DESCRIPTION
**Pull request by Matthalp**
Mixed caps variable names is a golang best practice [1].

[1] https://golang.org/doc/effective_go.html#mixed-caps

### CODE REVIEW

**Comment by Matthalp** [comment]
Broken up per review in #19016.

**Comment by Matthalp** [comment]
@karalabe It looks like this PR was not assigned a reviewer. Would you be able to do that?

File: cmd/swarm/config.go [hunk 1]
```diff
@@ -59,31 +59,31 @@ var (
...
-	SWARM_ENV_BOOTNODE_MODE           = "SWARM_BOOTNODE_MODE"
-	SWARM_ACCESS_PASSWORD             = "SWARM_ACCESS_PASSWORD"
-	SWARM_AUTO_DEFAULTPATH            = "SWARM_AUTO_DEFAULTPATH"
-	SWARM_GLOBALSTORE_API             = "SWARM_GLOBALSTORE_API"
-	GETH_ENV_DATADIR                  = "GETH_DATADIR"
+	SwarmEnvChequebookAddr       = "SWARM_CHEQUEBOOK_ADDR"
+	SwarmEnvAccount              = "SWARM_ACCOUNT"
+	SwarmEnvListenAddr           = "SWARM_LISTEN_ADDR"
+	SwarmEnvPort                 = "SWARM_PORT"
+	SwarmEnvNetworkId            = "SWARM_NETWORK_ID"
```
Comment (janos) [inline cmd/swarm/config.go]: SwarmEnvNetworkId should be SwarmEnvNetworkID.

Comment (Matthalp) [reply cmd/swarm/config.go]: Done. I also treated ENS and API (as suggested) as well as PSS.

**Comment by Matthalp** [comment]
Thanks for the review @janos. I believe I have addressed the initialisms throughout the code changes. Please let me know if I missed something.

File: cmd/swarm/config.go [hunk 2]
```diff
@@ -59,31 +59,31 @@ var (
...
+	SwarmEnvNetworkID            = "SWARM_NETWORK_ID"
+	SwarmEnvSwapEnable           = "SWARM_SWAP_ENABLE"
+	SwarmEnvSwapAPI              = "SWARM_SWAP_API"
+	SwarmEnvSyncDisable          = "SWARM_SYNC_DISABLE"
+	SwarmEnvSyncUpdateDelay      = "SWARM_ENV_SYNC_UPDATE_DELAY"
+	SwarmEnvMaxStreamPeerServers = "SWARM_ENV_MAX_STREAM_PEER_SERVERS"
+	SwarmEnvLightNodeEnable      = "SWARM_LIGHT_NODE_ENABLE"
+	SwarmEnvDeliverySkipCheck    = "SWARM_DELIVERY_SKIP_CHECK"
+	SwarmEnvENSAPI               = "SWARM_ENS_API"
+	SwarmEnvEnsAddr              = "SWARM_ENS_ADDR"
```
Comment (janos) [inline cmd/swarm/config.go]: SwarmEnvENSAddr

File: cmd/swarm/main.go [hunk 3]
```diff
@@ -76,8 +76,8 @@ var gitCommit string
 
 //declare a few constant error messages, useful for later error check comparisons in test
 var (
-	SWARM_ERR_NO_BZZACCOUNT   = "bzzaccount option is required but not set; check your config file, command line or environment variables"
-	SWARM_ERR_SWAP_SET_NO_API = "SWAP is enabled but --swap-api is not set"
+	SwarmErrNoBzzaccount = "bzzaccount option is required but not set; check your config file, command line or environment variables"
+	SwarmErrSwapSetNoApi = "SWAP is enabled but --swap-api is not set"
```
Comment (janos) [inline cmd/swarm/main.go]: SwarmErrSwapSetNoAPI

File: cmd/swarm/main.go [hunk 4]
```diff
@@ -76,8 +76,8 @@ var gitCommit string
 
 //declare a few constant error messages, useful for later error check comparisons in test
 var (
-	SWARM_ERR_NO_BZZACCOUNT   = "bzzaccount option is required but not set; check your config file, command line or environment variables"
-	SWARM_ERR_SWAP_SET_NO_API = "SWAP is enabled but --swap-api is not set"
+	SwarmErrNoBzzaccount = "bzzaccount option is required but not set; check your config file, command line or environment variables"
```
Comment (janos) [inline cmd/swarm/main.go]: SwarmErrNoBzzAccount

File: cmd/swarm/config.go [hunk 5]
```diff
@@ -59,31 +59,31 @@ var (
...
+	SwarmEnvCors                 = "SWARM_CORS"
+	SwarmEnvBootnodes            = "SWARM_BOOTNODES"
+	SwarmEnvPSSEnable            = "SWARM_PSS_ENABLE"
+	SwarmEnvStorePath            = "SWARM_STORE_PATH"
+	SwarmEnvStoreCapacity        = "SWARM_STORE_CAPACITY"
+	SwarmEnvStoreCacheCapacity   = "SWARM_STORE_CACHE_CAPACITY"
+	SwarmEnvBootnodeMode         = "SWARM_BOOTNODE_MODE"
+	SwarmAccessPassword          = "SWARM_ACCESS_PASSWORD"
+	SwarmAutoDefaultpath         = "SWARM_AUTO_DEFAULTPATH"
```
Comment (janos) [inline cmd/swarm/config.go]: SwarmAutoDefaultPath

File: cmd/swarm/config.go [hunk 6]
```diff
@@ -59,31 +59,31 @@ var (
...
+	SwarmGlobalstoreAPI          = "SWARM_GLOBALSTORE_API"
+	GethEnvDatadir               = "GETH_DATADIR"
```
Comment (janos) [inline cmd/swarm/config.go]: GethEnvDataDir

File: cmd/swarm/config.go (hunk 2 above)
Comment (Matthalp) [reply cmd/swarm/config.go]: Done.

File: cmd/swarm/config.go (hunk 5 above)
Comment (Matthalp) [reply cmd/swarm/config.go]: Done.

File: cmd/swarm/config.go (hunk 6 above)
Comment (Matthalp) [reply cmd/swarm/config.go]: Done.

File: cmd/swarm/main.go (hunk 4 above)
Comment (Matthalp) [reply cmd/swarm/main.go]: Made this `SwarmErrNoBZZAccount`.

File: cmd/swarm/main.go (hunk 3 above)
Comment (Matthalp) [reply cmd/swarm/main.go]: Done.

File: cmd/swarm/config.go [hunk 7]
```diff
@@ -59,31 +59,31 @@ var (
...
+	SwarmEnvSwapEnable           = "SWARM_SWAP_ENABLE"
+	SwarmEnvSwapAPI              = "SWARM_SWAP_API"
+	SwarmEnvSyncDisable          = "SWARM_SYNC_DISABLE"
+	SwarmEnvSyncUpdateDelay      = "SWARM_ENV_SYNC_UPDATE_DELAY"
+	SwarmEnvMaxStreamPeerServers = "SWARM_ENV_MAX_STREAM_PEER_SERVERS"
+	SwarmEnvLightNodeEnable      = "SWARM_LIGHT_NODE_ENABLE"
+	SwarmEnvDeliverySkipCheck    = "SWARM_DELIVERY_SKIP_CHECK"
+	SwarmEnvENSAPI               = "SWARM_ENS_API"
+	SwarmEnvENSAddr              = "SWARM_ENS_ADDR"
+	SwarmEnvCORS                 = "SWARM_CORS"
```
Comment (Matthalp) [inline cmd/swarm/config.go]: Noticed that `CORS` should also probably be capitalized. There are several variables related to CORS` in this package, but can address that on a later set of PR(s) that address initialisms in the code base.

File: swarm/fuse/swarmfs.go [hunk 8]
```diff
@@ -24,10 +24,10 @@ import (
 )
 
 const (
-	Swarmfs_Version = "0.1"
-	mountTimeout    = time.Second * 5
-	unmountTimeout  = time.Second * 10
-	maxFuseMounts   = 5
+	SwarmfsVersion = "0.1"
```
Comment (Matthalp) [inline swarm/fuse/swarmfs.go]: Should this be `SwarmFSVersion`?

File: swarm/fuse/swarmfs.go [hunk 9]
```diff
@@ -24,10 +24,10 @@ import (
...
+	mountTimeout   = time.Second * 5
+	unmountTimeout = time.Second * 10
+	maxFuseMounts  = 5
```
Comment (Matthalp) [inline swarm/fuse/swarmfs.go]: Should this be `maxFUSEMounts`?

File: swarm/fuse/swarmfs.go (hunk 8 above)
Comment (janos) [reply swarm/fuse/swarmfs.go]: 👍

File: swarm/fuse/swarmfs.go (hunk 9 above)
Comment (janos) [reply swarm/fuse/swarmfs.go]: 👍

**Comment by Matthalp** [comment]
@janos PTAL

**Pull request merged**